utils/win32/widelands.rc.cmake

```diff
@@ -21,9 +21,10 @@
 
 A ICON MOVEABLE PURE LOADONCALL DISCARDABLE "@CMAKE_CURRENT_SOURCE_DIR@/../utils/win32/WL.ico"
 
-//
-// TO CHANGE VERSION INFORMATION, EDIT  OPTIONS BELOW...
-//
+// To change version information, edit FILEVERSION and PRODUCTVERSION. Keep these two identical.
+// Version "A,B,C,D" translates to version "0x000A000B.0x000C000D",
+// e.g. for version "1.2" we need "0,1,0,2".
```

Comment:
I beleive this is wrong,
the current value of this attribute in the windows file properties is 0.21.0.1,
so the 4 16 bit Integers are all separated by a point in the file properties.
this leads to 1.2 translates to 1,2,0,0 Which in fact will be 1.2.0.0 in the properties I believe

Reply:
Okay changed the explanation. I have no windows system available now to check this so I had to rely on microsoft's poor documentation